cli/coredump/coredump.go

```diff
@@ -0,0 +1,130 @@
+package coredump
+
+import (
+	"bytes"
+	"context"
+	"embed"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/codeclysm/extract"
+	"github.com/tarantool/tt/cli/cmdcontext"
+)
+
+//go:embed scripts/*
+var corescripts embed.FS
+
+func readEmbedFile(fs embed.FS, path string) (string, error) {
+	content, err := fs.ReadFile(path)
+	if err != nil {
+		return "", err
+	}
+	return string(content), nil
+}
+
+// packs coredump into a tar.gz archive.
+func pack(coreName string) error {
+	ctx, err := readEmbedFile(corescripts, "scripts/tarabart.sh")
+	if err != nil {
+		return fmt.Errorf("There was some problem packing archive. \n"+
+			"Error: '%v'.\n", err)
+	}
+	data := []byte(ctx)
+	path, err := os.Getwd()
+	err = ioutil.WriteFile(path+"/tarabart.sh", data, 0777)
```

Comment:
Why `0777`? Why does everyone need to have access to this file?

Reply:
Can't we not create a file, but just pass prepared commands to the `stdin` of the `shell`? Probably the script settings in this case can be passed through environment variables.

Reply:
Changed rights, about shell, i have some problems with using it, if my way of executing scripts is not ideal, i will investigate further.